<?xml version="1.0"?>
<launch>
    <param name="maxlen"      type="double" value="30.0" />    
    <param name="minlen"      type="double" value="0.001" />    
    <param name="pcTopic"     type="str"    value="/ouster/points" />
    <param name="reflec_img"  type="str"    value="/ouster/reflec_image" />
    <param name="signal_img"  type="str"    value="/ouster/signal_image" />
    <param name="nearir_img"  type="str"    value="/ouster/nearir_image" />
    <param name="range_img"   type="str"    value="/ouster/range_image" />
    <param name="detection_BoundingBoxes"   type="str"    value="/yolov5/detections" />    
    
    <param name="x_resolution"     type="double" value="0.15" /> <!-- X resolution of lidar on image plane-->
    <param name="ang_Y_resolution" type="double" value="2.05" />  
    <node pkg="ouster_filtering"   type="ouster_filtering_node" name="ouster_filtering_node" output="screen"/>

<node type="rviz" name="rviz" pkg="rviz" args="-d $(find ouster_filtering)/rviz/ouster_filtering.rviz" />
 
</launch>
